proto/Cargo.toml

```diff
@@ -38,6 +38,7 @@ uuid = { workspace = true, features = ["serde"] }
 webauthn-rs-proto = { workspace = true }
 sshkey-attest = { workspace = true }
 sshkeys = { workspace = true }
+log = "0.4.25"
```

Comment:
Why are you adding the log crate? We have tracing already and it exposes a log interface.

Reply:
my IDE sometimes gets weird ideas (like adding crates) and I don't always check what its suggesting properly, sorry